test/functional/cmdLineTests/criu/criuScript.sh

```diff
@@ -23,22 +23,27 @@
 #
 
 echo "start running script";
-$2 -XX:+EnableCRIUSupport $3 -cp "$1/criu.jar" $4 $5 >testOutput 2>&1;
-if [ "$5" == "SingleCheckpoint" ] || [ "$5" == "TwoCheckpoints" ] || [ "$5" == "ThreeCheckpoints" ]
-then
-sleep 2;
-criu restore -D ./cpData --shell-job;
-fi
-if [ "$5" == "TwoCheckpoints" ] || [ "$5" == "ThreeCheckpoints" ]
-then
-sleep 2;
-criu restore -D ./cpData --shell-job;
-fi
-if [ "$5" == "ThreeCheckpoints" ]
-then
-sleep 2;
-criu restore -D ./cpData --shell-job;
+# the expected arguments are:
+# $1 is the TEST_ROOT
+# $2 is the JAVA_COMMAND
+# $3 is the JVM_OPTIONS
+# $4 is the MAINCLASS
+# $5 is the NUM_CHECKPOINT
+# $6 is the KEEP_CHECKPOINT
```

Comment:
@LongyuZhang is this needed? we have `checkPointJVM(CRIUSupport criu, Path path, boolean deleteDir)`

Reply:
The parameter KEEP_CHECKPOINT is currently used as the condition for both restore control and testOutput file; the deleteDir is more about the folder and files, but maybe not for whether restore or not? Thanks.